sys/include/net/ng_sixlowpan/frag.h

```diff
@@ -42,10 +42,14 @@ extern "C" {
 #define NG_SIXLOWPAN_FRAG_SIZE_MASK     (0x07ff)    /**< mask for datagram size */
 
 /**
- * @brief   General and 1st 6LoWPAN fragmentation header
+ * @brief   1st 6LoWPAN fragmentation header
  *
- * @see <a href="https://tools.ietf.org/html/rfc4944#section-5.1">
- *          RFC 4944, section 5.1
+ * @note    Before the implementation determines if the header is a 1st or a
+ *          subsequent 6LoWPAN fragmentation header, this it is casted to this
+ *          data type.
```

Comment:
"[…] it is assumed to be this data type." is a more precise wording IMHO than "[…] it is casted to this data type."

But if you obfuscate this fact anyways by removing the "General and" above this note would be more confusing to me than helpful. So *please* leave the "General and" in there, especially, since this data type is called for this very reason `ng_sixlowpan_frag_t` and not `ng_sixlowpan_frag_1st_t`.